### ISSUE TITLE
Group name in notification

### ISSUE THREAD

**Issue by OutbackMatt** [Ylianst/MeshCentral#726]
Can I get the group name + the machine name in the notifications that show up on the right...?

I work with Medical Centres, and they all have a nurse room, reception and consult room 1 etc
If this could show groups as well, that would help me know which nurse computer just logged in

I'd guess that I can work around by making the computer name more verbose

**Comment by Ylianst** [comment]
Good request, I just got it done and it will be in the next published release. However, Bryan needs to fix some unrelated problem and so, I am holding the release until tomorrow. I added this as an option, but I could make it always present if people want it that way. Feedback appreciated.

![MC2-NotifyGroupName](https://user-images.githubusercontent.com/1319013/70762944-49e5e080-1d07-11ea-9b9c-2d6234e5aca6.png)
